app/src/main/java/rs/ruffle/PlayerActivity.kt

```diff
@@ -123,9 +123,16 @@ class PlayerActivity : GameActivity() {
                     item.setChecked(true)
                 }
             }
+            val resourceIdExit: Int = items.size + 1
```

Comment:
I don't think this has anything to do with resources...? :thinking:
As in, these things:
https://developer.android.com/guide/topics/resources/providing-resources
https://developer.android.com/reference/android/content/res/Resources

Reply:
Perhaps item?